### PR TITLE
Create Terminado guide

### PR DESCRIPTION
**Pull request by staticfloat**

### CODE REVIEW

File: _docs/guides/terminado.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    </script>
+  </body>
+</html>
+```
+
+Save this file as `index.html` alongside `app.py` in the root directory of the `Xterm.js` repository, run the application via `python app.py` and connect to `http://localhost:8010` to see a terminal created within your browser.
+
+## Terminado communications protocol
+
+The difference between the `terminadoAttach()` function provided by the `terminado` plugin and the `attach()` function provided by the `attach` plugin are minor. `Terminado` simply requires messages declare the type of data they carry by sending (for example) `['stdin', stdin_data]` instead of `stdin_data` over the websockets. The differences between the two plugins can be easily seen by diff'ing the two plugins' javascript files in the `addons` directory of the `Xterm.js` repository.
```
Comment (parisk) [inline _docs/guides/terminado.md]: Can yo rename **plugin** to **addon** please?

Comment (staticfloat) [inline _docs/guides/terminado.md]: You'd think that the fact that I explicitly mention the `addon` directory would be enough to clue me in.  ;)

Done.

File: _docs/guides/terminado.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+title: Connecting to Terminado
+category: Guides
+---
+
+Xterm.js can connect to backends other than the default node.js backend, as an example we will build a simple [Terminado](https://github.com/takluyver/terminado) application, and hook up to it with `Xterm.js`. `Terminado` is a python/twisted websocket host for terminal applications.
```
Comment (takluyver) [inline _docs/guides/terminado.md]: Nitpick: Terminado is based on the tornado async machinery, not twisted.

Comment (parisk) [inline _docs/guides/terminado.md]: @staticfloat please resolve this comment and we should be ready to go 😊 .

**Comment by staticfloat** [comment]
Fixed and squashed.

**Comment by parisk** [comment]
Great. Thanks for this contribution!

**Pull request merged**